src/esp/io/json.h

```diff
@@ -56,8 +56,8 @@ bool jsonIntoVal(CORRADE_UNUSED const JsonDocument& d,
 }  // jsonIntoVal template definition
 
 /**
- * @brief Check passed json doc for existence of passed @ref tag as
- * float. If present, populate passed @ref val with value. Returns whether tag
+ * @brief Check passed json doc for existence of passed tag as
+ * float. If present, populate passed val with value. Returns whether tag
```

Comment:
FYI the syntax for referencing function params is `@p tag`  / `@p val`. Unfortunately Doxygen doesn't check that it's actually a real parameter :disappointed: (OTOH, the Python docs do check that)

Reply:
ty, was wondering about that.